app/views/posts/show.html.erb

```diff
@@ -1,32 +1,26 @@
-<p class="notice text-green-700"><%= notice %></p>
-<div class="m-5vw">
-  <div class="border-solid p-20 flex-container flex-col mb-2vh">
+
+<div class="bg-white p-3 mb-3  w-50 mx-auto">
+  <div class="row">
     <div>
-      <h3>
-      <%= @post.title %> by <%= @post.author.name %>
-      </h3>
-      <%= render partial: "comm_like", locals:{post: @post}  %>
+      <h4> <%= @post.title %> by <%= @post.author.name %></h4>
+    </div>
+    <hr>
+    <div class="card-body">
+    <p><%= @post.text %></p>
+      <%= render partial: "comm_like", locals:{post: @post, user_id: @user_id, post_id: @post.id}  %>
+      <% if @post.comments_counter > 0 %>
+      <hr>
+      <div class="rounded p-3 bg-light">
+        <%= render partial: "comment", locals:{comments: @post.comments}  %>
       </div>
-    <div>
-      <p><%= @post.text %></p>
+       <% end %>
+    </div>
+    <div class="card-footer">
+      <%= link_to "Comment", new_user_post_comment_path({post_id: @post.id, user_id: @post.author_id}), method: :get, class:"text-primary m-1"%>
+      <%= link_to "Like", "/users/#{@post.author_id}/posts/#{@post.id}/likes", method: :post, class:"text-info m-1"%>
+      <% if can? :destroy, @post %>
+      <%= link_to "Delete Post", "/delete/post/#{@post.id}", method: :get, class: "text-danger m-1" %>
+      <% end %>
```

Comment:
- [ ] After creating a post as the current user, I was redirected to the post show page, however, when I clicked on `delete post` to delete the post I created, the post was not deleted till I clicked the second time before it gets deleted. Kindly fix this.  Check this [video](https://www.loom.com/share/50abd75a4efc4e7483e035dcf06919c2) I recorded for more details concerning the bug.

Reply:
Thank you. This is fixed.